src/sentry/uptime/endpoints/organization_uptime_stats.py

```diff
@@ -206,10 +211,14 @@ def _format_response(
         for timeseries in response.result_timeseries:
             subscription_id = timeseries.group_by_attributes["uptime_subscription_id"]
             status = timeseries.group_by_attributes["check_status"]
+            incident_status = timeseries.group_by_attributes["incident_status"]
+
+            if status == "failure" and incident_status == str(IncidentStatus.IN_INCIDENT.value):
```

Comment:
i'm wondering if this will end up biting us elsewhere. could we handle this logic on the frontend?

Reply:
we could. What do you think makes sense for the response to look like in that case?

Reply:
to just return incident status as its own column and not modify status at all

Reply:
why can't it just be 
```
ts, {failure: int, missed_window: int, success: int, incident_status: int}]
```

Reply:
You're right though, we won't end up representing this as part of the checks table (part of the reason I was thinking it would be useful to have a serializer for the checks so we could put this translation there)

Reply:
how could there be multiple checkin values for the same timestamp?

Reply:
Because it's a bucket of timestamps right. so some might be a part of an incident others not.

Reply:
👍🏼